tests/test_extrapolation.py

```diff
@@ -0,0 +1,69 @@
+import pandas as pd
+
+from xgbse import XGBSEDebiasedBCE
+from xgbse.extrapolation import extrapolate_constant_risk
+from xgbse.non_parametric import get_time_bins, calculate_kaplan_vectorized
+from tests.data import get_data
+from tests.test_survival_curves import monotonicity, between_01
+
+(
+    X_train,
+    X_test,
+    X_valid,
+    T_train,
+    T_test,
+    T_valid,
+    E_train,
+    E_test,
+    E_valid,
+    y_train,
+    y_test,
+    y_valid,
+    features,
+) = get_data()
+
+
+# generating Kaplan Meier for all tests
+
+time_bins = get_time_bins(T_train, E_train, 100)
+
+mean, high, low = calculate_kaplan_vectorized(
+    T_train.values.reshape(1, -1), E_train.values.reshape(1, -1), time_bins
+)
+
+km_survival = pd.concat([mean] * len(y_train))
+km_survival = km_survival.reset_index(drop=True)
```

Comment:
Do we use this dataframe anywhere in the tests file? If not, shouldn't we remove it?

Reply:
We use it in line 81, to assert that c-index of km is 0.5

Reply:
Is it in this .py file? test_extrapolation.py goes up to line 69..

Reply:
You're talking about test_metrics.py file, right? I don't see in this file any usage for km_survival.

Reply:
you're right, we don't use it here, will remove